### DISCUSSION TITLE
您好，请问支持达梦和人大金仓数据库的连接嘛？

### DISCUSSION THREAD

**Discussion by linaotian-vv** [dbeaver/dbeaver#11861] [General]
#### System information: 
- Operating system (distribution) and version
- DBeaver version
- Java version 
- Additional extensions

#### Connection specification:
- Database name and version
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)?

**Comment by ShadelessFox** [comment]
Hello @linaotian-vv,

Please fill the issue form using plain English.

**Comment by linaotian-vv** [comment]
Can you provide the connection between DM database and Kingbase database？

**Comment by linaotian-vv** [comment]
These two databases are used by many people in China

**Comment by uslss** [comment]
Dameng DB support feature request #10133

**Comment by uslss** [comment]
You can also create a feature request about Kingbase support
